src/components/Form.js

```diff
@@ -135,8 +135,34 @@ function Form(props) {
 
             // Validate the input for html tags. It should supercede any other error
             _.each(trimmedStringValues, (inputValue, inputID) => {
+
+                const m = inputValue.search(CONST.VALIDATE_FOR_HTML_TAG_REGEX);
```

Comment:
Can we get more descriptive name for this variable? `m` seems very general

Reply:
changed name to workspaceRegex